<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeBandForm</class>
 <widget class="QWidget" name="TimeBandForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Band Details</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="edtName"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Bill Method:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbFlatFee">
             <property name="text">
              <string>Flat Fee</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbIncPercent">
             <property name="text">
              <string>Inrement By Percentage</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0,1,0">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Base Price:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="edtBasePrice">
             <property name="maximum">
              <double>999999.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Base Duration:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="edtBaseDuration">
             <property name="maximum">
              <number>60</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_29">
             <property name="text">
              <string>Secs</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QCheckBox" name="cbLock">
           <property name="text">
            <string>Lock Time Band</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Dur. 1:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="sbD1">
             <property name="maximum">
              <number>60</number>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>% 1:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QSpinBox" name="sbP1">
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Dur. 2:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="sbD2">
             <property name="maximum">
              <number>60</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>% 2:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QSpinBox" name="sbP2">
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Dur. 3:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="sbD3">
             <property name="maximum">
              <number>60</number>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>% 3:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QSpinBox" name="sbP3">
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Dur. 4:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="sbD4">
             <property name="maximum">
              <number>60</number>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>% 4:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QSpinBox" name="sbP4">
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Dur. 5:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="sbD5">
             <property name="maximum">
              <number>60</number>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>% 5:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QSpinBox" name="sbP5">
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Dur. 6:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="sbD6">
             <property name="maximum">
              <number>60</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>% 6:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QSpinBox" name="sbP6">
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Dur. 7:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="sbD7">
             <property name="maximum">
              <number>60</number>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>% 7:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QSpinBox" name="sbP7">
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Dur. 8:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="sbD8">
             <property name="maximum">
              <number>60</number>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>% 8:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QSpinBox" name="sbP8">
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>Dur. 9:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="sbD9">
             <property name="maximum">
              <number>60</number>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_25">
             <property name="text">
              <string>% 9:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QSpinBox" name="sbP9">
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_26">
             <property name="text">
              <string>Dur. 10:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="sbD10">
             <property name="maximum">
              <number>60</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>% 10:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QSpinBox" name="sbP10">
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_27">
             <property name="text">
              <string>Dur. 11:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="sbD11">
             <property name="maximum">
              <number>60</number>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="label_28">
             <property name="text">
              <string>% 11:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QSpinBox" name="sbP11">
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>Dur. 12:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="sbD12">
             <property name="maximum">
              <number>60</number>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>% 12:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QSpinBox" name="sbP12">
             <property name="maximum">
              <number>100</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Daypart</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="vlDaypart"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
